schemas/schema.json

```diff
@@ -2677,6 +2684,9 @@
                         "pattern": "^[a-zA-Z0-9\\&\\ \\-\\.\\,\\/\\_\\(\\)\\']+$",
                         "type": "string",
                         "description": "The Name of the Buyer"
+                    },
+                    "vatNumber": {
+                        "$ref": "#/components/schemas/VATNumber"
```

Comment:
might be worth adding a naive regex pattern for VAT eg.

`"^[A-Z]{2}\\d{9}$"`

Reply:
Addressed in https://github.com/RecyclingAssociation/traqa/pulls as this PR is closed